INSTALL.md

```diff
@@ -7,11 +7,11 @@ You can either use Docker Compose to install Node and PostgreSQL in containers,
 1. Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/).
 1. Clone this repository.
 1. Create a CSV file with a single column called "ip" with your meter IP addresses and copy it into the directory where the project resides.
-1. Set up Node environment and the database by running ```docker-compose run --rm web src/scripts/init.sh <csv file>``` in the main directory. 
+1. Set up Node environment and the database by running ```docker-compose run --rm web src/scripts/init.sh <csv file> --default-user``` in the main directory. 
```

Comment:
Recommend against setting up a default user for development users. Perhaps notify them that it is an option, but default to forcing them to make a user.